### PR TITLE
Basic Prometheus Metrics

### PR DESCRIPTION
**Pull request by goern**
Exporting per method, per endpoint Prometheus Metrics.

### CODE REVIEW

File: Pipfile.lock [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
-            "platform_release": "4.15.14-300.fc27.x86_64",
-            "platform_system": "Linux",
-            "platform_version": "#1 SMP Thu Mar 29 16:13:44 UTC 2018",
+            "platform_release": "17.5.0",
+            "platform_system": "Darwin",
+            "platform_version": "Darwin Kernel Version 17.5.0: Mon Mar  5 22:24:32 PST 2018; root:xnu-4570.51.1~1/RELEASE_X86_64",
             "python_full_version": "3.6.5",
             "python_version": "3.6",
-            "sys_platform": "linux"
+            "sys_platform": "darwin"
```
Comment (durandom) [inline Pipfile.lock:17]: @goern is this a problem? Looks like you've frozen you pip env on a mac but we'll deploy on linux

File: wsgi.py [hunk 2]
```diff
@@ -24,24 +24,25 @@
...
 
 import requests
 
 from prometheus_client import CONTENT_TYPE_LATEST
-from prometheus_client import Counter
-from prometheus_client import Histogram
-from prometheus_client import core
-from prometheus_client import generate_latest
+from prometheus_client import Counter, Histogram
+from prometheus_client import core, generate_latest
```
Comment (durandom) [inline wsgi.py:34]: seems to be good for such a small file, but in general I guess you would move all prometheus stuff into its own package to not to pollute the namespace here. Eg. `CONTENT_TYPE_LATEST`
Or just `import prometheus` and reference with a FQN like `prometheus.CONTENT_TYPE_LATEST`

**Review by durandom — APPROVED** [review]
👍 LGTM

File: Pipfile.lock (hunk 1 above)
Comment (goern) [reply Pipfile.lock:17]: thats not a problem to the S2I on OpenShift GNU/Linux

**Pull request merged**